cmd/kogito/command/shared/resource_checks.go

```diff
@@ -83,7 +83,8 @@ func checkKogitoOperatorExists(kubeCli *client.Client, namespace string) (bool,
 	}
 
 	if operatorDeployment.Status.AvailableReplicas == 0 {
-		return true, fmt.Errorf("Kogito Operator seems to be created in the namespace '%s', but there's no available pods replicas deployed ", namespace)
+		log.Warnf("Kogito Operator seems to be created in the namespace '%s' but there's no available pods replicas deployed ", namespace)
```

Comment:
I did not get why this was changed to a warn ?
There should be an error if no available replica or ?